### DISCUSSION TITLE
ObjectBox Local DB

### DISCUSSION THREAD

**Discussion by milanmiff** [wailsapp/wails#2382] [General]
Is it possible to build for production and use ObjectBox as local storage?
I can't find any documentation on how to include storage in the build...

**Comment by stevealliance** [comment]
@milanmiff  In my case I've used Lungo and it's working fine for saving data in your local machine.
https://github.com/256dpi/lungo#example

**Comment by stevealliance** [comment]
@milanmiff Yes you can, no real documentation just use the side card pattern I've done the same with Lungo that is just a Mongo DB file manager (document)
